kpatch-build/create-diff-object.c

```diff
@@ -3041,11 +3067,11 @@ void kpatch_write_output_elf(struct kpatch_elf *kelf, Elf *elf, char *outfile)
 }
 
 struct arguments {
-	char *args[4];
+	char *args[5];
```

Comment:
Interestingly, this didn't segfault when it *should have*. Yay for undefined C behavior!